Access API/AccessAPI/BLL/SearchBLL.cs

```diff
@@ -1,23 +1,26 @@
 ﻿using System;
+using System.Collections.Generic;
 using Access_API.DAL;
 
 namespace Access_API.BLL
 {
     public class SearchBLL
     {
-        public string SearchBll(string words, int? sourceId, string? author, int? categoryId,
-            DateTime? beforeDate, DateTime? afterDate)
+        public string SearchBll(string words, List<long>? sourceIds, List<string>? authors, List<int>? categoryId,
+            DateTime? beforeDate, DateTime? afterDate, int? limit, int? offset)
```

Comment:
```suggestion
        public string SearchBll(string words, List<long> sourceIds, List<string> authors, List<int> categoryId,
            DateTime? beforeDate, DateTime? afterDate, int? limit, int? offset)
```
Jeg vil foreslå at bruge tomme lister frem for null her